client/src/components/EventView/index.js

```diff
@@ -28,18 +50,26 @@ const EventView = () => {
   return (
     <div>
       <AppBar color={eventColor ?? theme1} />
-      <Layout>
-        Event View
-        <Location address="Provo UT" />
+      <Layout><DisplayStle>
+        <RSVPFloating></RSVPFloating></DisplayStle>
```

Comment:
The component you're looking for is in the Common folder, and it is called `<ActionPrompt>`. That component should replace the RSVPFloating